packages/circuits/test/utils.ts

```diff
@@ -266,7 +267,10 @@ const genStartTransitionCircuitInput = (
     return crypto.stringifyBigInts(circuitInputs)
 }
 
-const bootstrapRandomUSTree = async (): Promise<crypto.SparseMerkleTree> => {
+const bootstrapRandomUSTree = async (): Promise<{
+    userStateTree
+    reputationRecords
```

Comment:
I think we should add types to parameters and void to use `any`.

Reply:
what if we define type here?
```typescript
type UserStates = {
    userStateTree: SparseMerkleTree,
    reputationRecords: { [key: string]: IReputation }
}
```

Reply:
Yes, we should avoid you `any`